src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -51,6 +58,13 @@ class MemberPermission(OrganizationPermission):
     }
 
 
+class OrganizationGroupSearchViewGetSerializer(serializers.Serializer[None]):
+    visibility = serializers.MultipleChoiceField(
+        choices=[v[0] for v in GroupSearchViewVisibility.as_choices()],
```

Comment:
according to docs you should just be able to pass in `as_choices()` directly without having to pluck out the keys https://www.django-rest-framework.org/api-guide/fields/#multiplechoicefield